runtime/spacetime_nat.c

```diff
@@ -355,8 +354,8 @@ CAMLprim value caml_spacetime_save_event (value v_time_opt,
 }
 
 
-void save_trie (struct channel *chan, double time_override,
-                int use_time_override)
+static void save_trie (struct channel *chan, double time_override,
```

Comment:
This gets caught (correctly) by tool/check-symbol-names and shouldn't be global without being prefixed `caml_`